map/framework.cpp

```diff
@@ -189,9 +188,9 @@ Framework::Framework()
   : m_bmManager(*this)
   , m_fixedSearchResults(0)
 {
-  m_activeMaps.reset(new ActiveMapsLayout(*this));
-  m_globalCntTree = storage::CountryTree(m_activeMaps);
-  m_storageBridge = make_unique_dp<StorageBridge>(m_activeMaps, bind(&Framework::UpdateCountryInfo, this, _1, false));
+//  m_activeMaps.reset(new ActiveMapsLayout(*this));
+//  m_globalCntTree = storage::CountryTree(m_activeMaps);
+//  m_storageBridge = make_unique_dp<StorageBridge>(m_activeMaps, bind(&Framework::UpdateCountryInfo, this, _1, false));
```

Comment:
What's the point in this comment? You've removed ActiveMapsLayout, remove this lines too )